### PR TITLE
add Baidu Kunlun XPU Card installation documents in Chinese/English

### PR DESCRIPTION
**Pull request by vslyu**
There are two Installation methods supported as following:
- Installation method one：pre-built Wheel package
- Installation method two：by compiling Paddle with Kunlun XPU support

### CODE REVIEW

File: doc/fluid/install/install_Kunlun_zh.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+o  ```wget https://fleet.bj.bcebos.com/kunlun/paddlepaddle-0.0.0-cp37-cp37m-linux_x86_64.whl```
+
+o  ```python3.7 -m pip install -U paddlepaddle-0.0.0-cp37-cp37m-linux_x86_64.whl ```
+
+Python2.7
+
+o  ``` wget https://fleet.bj.bcebos.com/kunlun/paddlepaddle-0.0.0-cp27-cp27mu-linux_x86_64.whl ```
+
+o  ```python2.7 -m pip install -U paddlepaddle-0.0.0-cp27-cp27mu-linux_x86_64.whl```
+
```
Comment (swtkiwi) [inline doc/fluid/install/install_Kunlun_zh.md]: 改为2.0rc1的包哈，paddlepaddle-2.0.0rc1-cp37-cp37m-linux_x86_64.whl，QA后续会编新的包

File: doc/fluid/install/install_Kunlun_en.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+o  ```wget https://fleet.bj.bcebos.com/kunlun/paddlepaddle-0.0.0-cp37-cp37m-linux_x86_64.whl```
+
+o  ```python3.7 -m pip install -U paddlepaddle-0.0.0-cp37-cp37m-linux_x86_64.whl ```
+
+Python2.7
+
+o  ``` wget https://fleet.bj.bcebos.com/kunlun/paddlepaddle-0.0.0-cp27-cp27mu-linux_x86_64.whl ```
+
+o  ```python2.7 -m pip install -U paddlepaddle-0.0.0-cp27-cp27mu-linux_x86_64.whl```
+
```
Comment (swtkiwi) [inline doc/fluid/install/install_Kunlun_en.md]: 版本号2.0.0rc1

File: doc/fluid/install/install_Kunlun_en.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
+# Paddle installation for machines with Kunlun XPU card
+
+## Installation method one：pre-built Wheel package
```
Comment (Heeenrrry) [inline doc/fluid/install/install_Kunlun_en.md]: Installation Method 1/Installation Method One, 专有名词这样更容易辨认

File: doc/fluid/install/install_Kunlun_en.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+Install the pre-built wheel package that supports Kunlun XPU. At present, this wheel package only supports the Ubuntu environment. For other environments, please choose Installation method two to installing from source code compilation.
```
Comment (Heeenrrry) [inline doc/fluid/install/install_Kunlun_en.md]: Installation Method 2

File: doc/fluid/install/install_Kunlun_en.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+or
+
+ ```pip3 uninstall paddlepaddle ```
+
+In addition, if there are environmental problems with the pre-built wheel package that supports Kunlun XPU card, it is recommended to use Installation method two  to compile the package that supports Kunlun XPU.
+
+
+
+## Installation method two：by compiling Paddle with Kunlun XPU support
```
Comment (Heeenrrry) [inline doc/fluid/install/install_Kunlun_en.md]: Installation Method 2

File: doc/fluid/install/install_Kunlun_zh.md (hunk 1 above)
Comment (Heeenrrry) [reply doc/fluid/install/install_Kunlun_zh.md]: fleet.bj.bcebos.com -->  paddle-wheel.bj.bcebos.com 其他包在这，还是统一更好

File: doc/fluid/install/install_Kunlun_en.md (hunk 5 above)
Comment (vslyu) [reply doc/fluid/install/install_Kunlun_en.md]: done

File: doc/fluid/install/install_Kunlun_en.md (hunk 4 above)
Comment (vslyu) [reply doc/fluid/install/install_Kunlun_en.md]: done

File: doc/fluid/install/install_Kunlun_en.md (hunk 3 above)
Comment (vslyu) [reply doc/fluid/install/install_Kunlun_en.md]: done

File: doc/fluid/install/install_Kunlun_en.md (hunk 2 above)
Comment (vslyu) [reply doc/fluid/install/install_Kunlun_en.md]: done

File: doc/fluid/install/install_Kunlun_zh.md (hunk 1 above)
Comment (vslyu) [reply doc/fluid/install/install_Kunlun_zh.md]: paddle-wheel.bj.bcebos.com bos下面也有很多子文件夹，文件夹名字还是用昆仑么～

Comment (Heeenrrry) [reply doc/fluid/install/install_Kunlun_zh.md]: 还用kunlun吧，好维护一点

Comment (vslyu) [reply doc/fluid/install/install_Kunlun_zh.md]: done

**Review by Heeenrrry — APPROVED** [review]
LGTM

**Pull request merged**